mne/viz/_brain/_brain.py

```diff
@@ -2368,6 +2372,76 @@ def add_foci(self, coords, coords_as_verts=False, map_surface=None,
                                   opacity=alpha, resolution=resolution)
             self._renderer.set_camera(**views_dicts[hemi][v])
 
+    @fill_doc
+    def add_sensors(self, info, trans=None, ch_types=None, colors=None,
+                    scales=None, alphas=None):
+        """Add an ROI label to the image.
+
+        Parameters
+        ----------
+        info : mne.Info
+            The measurement information with sensor locations.
+        %(trans)s
+        ch_types : dict
+            Channel type keyword arguments to be passed to
+            :func:`mne.pick_types`.
+        colors : str | dict
+            The color of the sensors. If dict, channel names or types can be
+            keys with values of matplotlib colors. Default is all sensors
+            yellow.
+        scales : float | dict
+            The size of the sensors. If dict, channel names or types can be
+            keys with values for the size in mm. Default is all sensors 5 mm.
+        alphas : float | dict
+            The opacity of the sensors or, if dict, each sensor by name or
+            type. Default is all sensors 0.5 opacity.
```

Comment:
This API (and code) should be refactored with `mne.viz.plot_alignment` to the extent possible. We many options for plotting sensor types there:

- MEG with helmet, or just as sensors
- EEG as original, or projected to the scalp
- fNIRS in many configurations

These are all useful for different use cases. If we're going to have a generic `plot_sensors` function, we should port that functionality (for usability) and API (for consistency) over as much as possible.

Reply:
What would you think about making fNIRS it's own "add" function? It seems different enough and all the overloads on dictionaries in `plot_alignment` seem really confusing to me. I think both on the user and developer side it's easier to understand and maintain if the functions are a bit smaller in scope.

Reply:
To me the API `eeg=..., meg=..., fnirs=...` is simple enough. I'd rather have this than `add_sensors` and `add_sensors_fnirs`, to me it makes more sense to have the fnirs options in a keyword like this. The API for `fnirs` was hashed out with fNIRS people in the last year or two over a few PRs, so it should be "good enough" for them. But maybe  @rob-luke has thoughts / regrets ...

Reply:
That's persuasive enough for me, I'll take a quick shot at adding that and then we can either merge or pause on this for a while as I was waiting for the BEM refactoring to be done so I think I'll go back to getting going on the GUI.

Reply:
I would prefer not to have a plot_sensors_fNIRS function, and instead have a fNIRS argument like we currently do. I think the current system works fine, has there been some issues with the current API?

I'm happy to test any new code with fNIRS data for you. Just ping me.

Reply:
> I think both on the user and developer side it's easier to understand and maintain if the functions are a bit smaller in scope.

I disagree. From a user standpoint, I think it would be confusing if some types work with plot_sensors, but others don't and require a special function. Users don't see the internal wrangling, and just expect a function to work with data regardless if it's eeg, meg, fnirs.

Reply:
This was more in regards to adding the head separately and the sensors separately etc. I already pushed to put fNIRS in

Reply:
Awesome, appreciate it!